content/id/docs/tasks/_index.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Tugas-tugas (_Tasks_)
+main_menu: true
+weight: 50
+content_template: templates/concept
+---
+
+{{< toc >}}
+
+{{% capture overview %}}
+
+Bagian dokumentasi Kubernetes ini berisi halaman-halaman yang perlihatkan 
+bagaimana melakukan setiap tugas (_task_). Halaman tugas menunjukkan cara melakukan
+satu hal saja, biasanya dengan memberikan urutan langkah pendek.
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Antarmuka Pengguna Berbasis Web (_Dashboard_)
+
+Menyebarkan dan mengakses Dashboard antarmuka pengguna berbasis web untuk 
```

Comment:
```suggestion
Melakukan _deploy_ dan mengakses dasbor berbasis web untuk 
```
to make it easier to understand in Bahasa, wdyt?

Reply:
Ok. But, I would like to change `dasbor` into `dasbor (_dashboard_)` to link with the title.